Makefile

```diff
@@ -2,41 +2,20 @@ GO ?= go
 GOFLAGS := -v
 PKG := ./...
 TESTS := ".*"
-GOIMPORTS := ../../../../bin/goimports
-GOLINT := ../../../../bin/golint
-GLOCK := ../../../../bin/glock
-BINDATA := ../../../../bin/go-bindata
+GOIMPORTS := goimports
+GOLINT := golint
```

Comment:
golint has been deprecated. We could add it to the module file, but it's probably better to remove references to it.

Reply:
Sounds good - I've removed the links to GOLINT too.